architecture/additional_concepts/networking.adoc

```diff
@@ -94,32 +97,166 @@ round-robin between each pod.
 
 [[network-plugins]]
 
-== Network Plugins
+== Network Plug-ins
 
-{product-title} supports the same plugin model as Kubernetes for networking pods.
+{product-title} supports the same plug-in model as Kubernetes for networking pods. 
 ifdef::openshift-enterprise,openshift-origin[]
-The following network plugins are currently supported by {product-title}.
+The following network plug-ins are currently supported by {product-title}.
 endif::[]
 
 [[openshift-sdn]]
 
 === {product-title} SDN
 
-{product-title} deploys a software-defined networking (SDN) approach for connecting
-pods in an {product-title} cluster. The {product-title} SDN connects all
-pods across all node hosts, providing a unified cluster network.
+{product-title} deploys a software-defined networking (SDN) approach for
+connecting pods in an {product-title} cluster. The {product-title} SDN connects
+all pods across all node hosts, providing a unified cluster network.
 
 {product-title} SDN is automatically installed and configured as part of the
-Ansible-based installation procedure.  Further administration should not
-be required; however, xref:sdn.adoc#architecture-additional-concepts-sdn[further details on the design and
-operation of {product-title} SDN] are provided for those who are curious or
-need to troubleshoot problems.
+Ansible-based installation procedure. See the
```

Comment:
I thin this one is OK as is. It's linking to the more contextual sdn stuff. Linking to the install stuff would change the meaning.